pkg/aws/core_util.go

```diff
@@ -68,6 +69,24 @@ func decodeProviderSpecAndSecret(machineClass *v1alpha1.MachineClass, secret *co
 	return providerSpec, nil
 }
 
+// disableSrcAndDestCheck disbales the SrcAndDestCheck for NAT instances
+func disableSrcAndDestCheck(svc ec2iface.EC2API, instanceID *string) error {
+
+	srcAndDstCheckEnabled := &ec2.ModifyInstanceAttributeInput{
+		InstanceId: instanceID,
+		SourceDestCheck: &ec2.AttributeBooleanValue{
+			Value: pointer.BoolPtr(false),
+		},
+	}
+
+	_, err := svc.ModifyInstanceAttribute(srcAndDstCheckEnabled)
+	if err != nil {
+		return err
+	}
+	klog.V(3).Infof("Successfully disabled Source/Destination check on NAT instance %s.", *instanceID)
```

Comment:
Minor log fix. This is a normal VM feature. NAT is only a use-case of this feature.
```suggestion
	klog.V(3).Infof("Successfully disabled Source/Destination check on instance %s.", *instanceID)
```